src/vmm/src/persist.rs

```diff
@@ -152,7 +157,7 @@ pub enum CreateSnapshotError {
 }
 
 /// Snapshot version
-pub const SNAPSHOT_VERSION: Version = Version::new(5, 0, 0);
+pub const SNAPSHOT_VERSION: Version = Version::new(6, 1, 0);
```

Comment:
We only update the major version at every change.

```suggestion
pub const SNAPSHOT_VERSION: Version = Version::new(6, 0, 0);
```